tooling/operation/get-parameters-as-json-schema.js

```diff
@@ -375,7 +390,12 @@ function getRequestBody(operation, oas) {
         }
 
         Object.keys(oas.components[componentType]).forEach(schemaName => {
-          components[componentType][schemaName] = constructSchema(oas.components[componentType][schemaName]);
+          components[componentType][schemaName] = constructSchema(
+            oas.components[componentType][schemaName],
+            undefined,
+            undefined,
```

Comment:
`currentLocation` should probably always be a string.

```suggestion
           '',
```